tests/test_debug/snapshot_main_frame.py

```diff
@@ -0,0 +1,18 @@
+"""
+This Python module is designed for testing the snapshot function's ability to return
+variables from the __main__ stack frame, particularly globally defined variables.
+
+This module is intended exclusively for testing purposes and should not be used for any other purpose.
+"""
+import json
+
+from python_ta.debug.snapshot import snapshot
+
+# globally defined variables
+team_lead = "David Liu"
+SDS_projects = ["PyTA", "MarkUs", "Memory Models"]
+team_num = 9
+
+global_vars = snapshot()[0]
```

Comment:
Keep things simple and don't store this value in a variable (since it will also appear as a global variable), instead just convert and print it directly